react-frontend/src/components/Community/communityCard.js

```diff
@@ -11,7 +12,9 @@ function CommunityCard({ title, description, uid }) {
   return (
     <CommunityCardStyled to={`/communities/${uid}`}>
       <CommunityCardHeader>{title}</CommunityCardHeader>
-      <CommunityCardDescription>{description}</CommunityCardDescription>
+      <CardClickableBody>
```

Comment:
Why is this being changed?  I don't see it being referenced in the zenhub QA cards or in the PR description

Reply:
Idea here was to introduce a **body** across all cards that would define the 24px padding and rounded bottom corners in one place, into which various other items can fit (e.g. titles, subtitles, descriptions, avatars, tags) based on the variations. 

I've manually tested card styles across the site and nothing is broken now. 

In a future, separate issue, I'd like to tackle a refactor of card components -- I think the number of styles can be reduced.